ntoskrnl/cc/view.c

```diff
@@ -71,10 +71,10 @@ ULONG CcTotalDirtyPages = 0;
 LIST_ENTRY CcDeferredWrites;
 KSPIN_LOCK CcDeferredWriteSpinLock;
 LIST_ENTRY CcCleanSharedCacheMapList;
-#ifndef _MSC_VER
+#if !defined(_MSC_VER) || _MSC_VER >= 1900
 LARGE_INTEGER CcIdleDelay = {.QuadPart = (LONGLONG)-1*1000*1000*10};
 #else
-LARGE_INTEGER CcIdleDelay = {(LONGLONG)-1*1000*1000*10};
+LARGE_INTEGER CcIdleDelay = { { -1*1000*1000*10, -1 } };
```

Comment:
And the day you try to compile for an architecture where LowPart and HighPart are reverted, you get BUGBUG!

Reply:
We could create a macro:
```#define RTL_MAKE_LARGE_INTEGER(low, high)```
that would correctly assign the members, and
```#define RTL_LARGE_INTEGER(longlong)```
that would basically do what is done above in the code. That would allow using these macros everywhere else where we have to otherwise use such #ifdefs .

Reply:
Uhm... NT is little-endian only, so that won't happen. Literally everything would break, not just this.

Reply:
That said, if you have a better idea, I'm open to it. I don't personally care that much about keeping vs2010 working, as you already know. ;P

Reply:
I like the macro idea. It could do exactly what's done here, and `C_ASSERT(FIELD_OFFSET(LARGE_INTEGER, LowPart) == 0)` to address the endianness concern

Reply:
We could call it RTL_CONSTANT_LARGE_INTEGER in the same fashion with RTL_CONSTANT_STRING

Reply:
Do we have any "custom reactos kernel stuffs".h file somewhere? Otherwise, where would you put that macro?

Reply:
NDK seems reasonable.